service-base/src/main/java/org/eclipse/hono/service/metric/MicrometerBasedMetrics.java

```diff
@@ -49,6 +51,10 @@
      * The name of the meter for authenticated connections.
      */
     public static final String METER_CONNECTIONS_AUTHENTICATED = "hono.connections.authenticated";
+    /**
+     * This metric is used to keep track of the connection duration of the authenticated devices to Hono.
+     */
+    public static final String METER_CONNECTIONS_DURATION_AUTHENTICATED = "hono.connections.duration.authenticated";
```

Comment:
For aesthetic reasons I'd prefer `hono.connections.authenticated.duration` so that the names form something like a hierarchy. WDYT?

Reply:
For me, it sounded easier to understand when `duration` is next to the `connections` namely `hono.connections.duration.authenticated`. I can revert to the old one if you prefer.

Reply:
That would be nice.

Reply:
I have reverted it to `hono.connections.authenticated.duration`. Please take a look again.